src/components/ApproveRequests.vue

```diff
@@ -65,14 +39,14 @@ export default {
     }
   },
   components: {
-    // Loading
+    RequestList
   },
   methods: {
     showContributions() {
       this.$store.dispatch('getUserContribution', this.accountNum)
     },
     /* eslint-disable */
-    async approveRequest(address, ethId, requestID) {
+    async approveRequest({ address, ethId, requestID }) {
```

Comment:
When working with cross-component methods, we're allowed a single argument to pass around. Hence, this becomes a single destructured parameter rather than three.

Reply:
Cool, I'm gonna have to take some notes.

Reply:
Ah that makes sense, very modular!